### ISSUE TITLE
Package dart

### ISSUE THREAD

**Issue by rxrbln** [rxrbln/t2sde#110]
https://dart.dev

**Comment by termdew** [comment]
I would like to work on this one, maybe it's a good issue for practicing packaging for T2 SDE.

**Comment by rxrbln** [comment]
sure, though dart might be more complex and thus difficult than the libreoffice bounty you I believe comments on earlier. Would you still consider the libreoffice reps bound? Because somehow your comment disappeared when I wanted to comment on it. Did you delete it?

**Comment by termdew** [comment]
Yeah, I deleted the comment, because I thought that one package would be better as a beginners task.If you say that Dart is more complex, then please assign me to the LibreOffice issue. Sorry for the confusion.Am 22.11.2023 um 21:02 schrieb René Rebe ***@***.***>:﻿
sure, though dart might be more complex and thus difficult than the libreoffice bounty you I believe comments on earlier. Would you still consider the libreoffice reps bound? Because somehow your comment disappeared when I wanted to comment on it. Did you delete it?

**Comment by ian-moody** [comment]
I think this can be closed as dart was first packaged as of commit 79f29a6.
